src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -325,6 +347,48 @@ private void generateIntervalsVCFFileFromAllShards() {
         intervalsVCFWriter.close();
     }
 
+    private void concatenateDenoisedCopyRatioFiles() {
```

Comment:
Can you move the `generateSegmentsVCFFileFromAllShards` method below to be just above this one?

Reply:
Done